frontend/src/views/setting/backup-account/webdav/index.vue

```diff
@@ -114,7 +114,8 @@ const handleClose = () => {
 };
 
 const toDoc = () => {
-    window.open(globalStore.docsUrl + '/user_manual/settings/#34-webdav-alist', '_blank', 'noopener,noreferrer');
+    const uri = globalStore.isIntl ? '#webdav-with-alist' : '#34-webdav-alist';
+    window.open(globalStore.docsUrl + '/user_manual/settings/' + uri, '_blank', 'noopener,noreferrer');
 };
 
 const onSubmit = async (formEl: FormInstance | undefined) => {
```

Comment:
The code looks generally correct but can be optimized. Here are some suggestions:

1. Directly append `uri` to `docsUrl`: No need for concatenation, just add it directly.
2. Use template literals instead of plus signs for string interpolation: Makes the code cleaner.

Optimized version:

```javascript
const handleClose = () => {};

const toDoc = () => {
    const uri = globalStore.isIntl ? '#webdav-with-alist' : '#34-webdav-alist';
    window.open(`${globalStore.docsUrl}/user_manual/settings/${uri}`, '_blank', 'noopener,noreferrer');
};

const onSubmit = async (formEl: FormInstance | undefined) => {
```

This change will make the opening of URLs more readable and maintainable.